Import automation: changing configs (#254)
datacommonsorg/data: import-automation/executor/README.md

gcloud app deploy
```

## Configuring the Executor

The executor has a number of customizble configurations listed in

> [app/configs.py](app/configs.py). Some have default values and some do not and
> need to be supplied. The Cloud Tasks tasks created by Cloud Build can pass the

Reviewer: ```suggestion
[app/configs.py](app/configs.py). Some have default values; the others need
explicit values. The Cloud Tasks tasks created by Cloud Build can pass the
```
